ansible/roles/test/files/ptftests/arista.py

```diff
@@ -240,18 +243,22 @@ def parse_logs(self, data):
 
             assert(events[-1][1] == 'Established')
 
+        # verify establishment time between v4 and v6 peer is not more than 20s
```

Comment:
what if both v4 and v6 started later than we expected?

Reply:
Based on data collected from multiple platforms and discussions with @yxieca, there will be an additional check that I will be adding later. That will ensure that for each peer, down to established time should not be more than 165sec.  The plan now is to watch the peer down to established time in the logs for a couple of weeks to see if that is the right number that we want to check.

Reply:
Each platform has its own cpu speed. So 165 sec could work for some platforms, but might fail for others
Also I don't think we need to add the bgp test here.
If we want bgp test, we can make it here: https://github.com/Azure/sonic-buildimage/tree/master/platform/vs/tests/bgp

Reply:
removed bgp time check. only fixing the log error in this PR